execution_engine_testing/tests/bin/disk_use.rs

```diff
@@ -0,0 +1,254 @@
+use std::{collections::HashMap, error::Error};
+
+use casper_execution_engine::storage::trie::Trie;
+use casper_hashing::Digest;
+use lmdb::{Cursor, Transaction};
+use tempfile::TempDir;
+
+use casper_engine_test_support::{
+    auction::{self},
+    transfer, LmdbWasmTestBuilder, DEFAULT_ACCOUNT_ADDR, MINIMUM_ACCOUNT_CREATION_BALANCE,
+};
+use casper_types::{account::AccountHash, bytesrepr::FromBytes, Key, StoredValue, U512};
+
+const TARGET_ADDR: AccountHash = AccountHash::new([127; 32]);
+
+fn transfer_disk_use(transfer_count: usize, purse_count: usize) -> Result<(), Box<dyn Error>> {
```

Comment:
Please add a comment describing what will be done and measured as part of this method call.